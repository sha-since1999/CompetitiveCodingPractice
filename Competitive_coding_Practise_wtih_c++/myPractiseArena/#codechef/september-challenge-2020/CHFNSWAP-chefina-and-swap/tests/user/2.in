10
9
14
19
1000
1243
213412
128392
89127323
12341
10930219
