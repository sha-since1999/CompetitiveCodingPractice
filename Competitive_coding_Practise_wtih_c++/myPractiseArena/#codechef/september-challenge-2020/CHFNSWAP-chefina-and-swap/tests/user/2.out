0
0
6
294
365
62508
37606
26104789
0
3201388